build/ci-build.yml

```diff
@@ -121,11 +129,36 @@ stages:
         pool:
           vmImage: 'ubuntu-16.04'
         steps:
-          - template: test/run-integration-tests.yml@templates
```

Comment:
Have to remove the template usage because we need both .NET Core SDK's and the installer task that's used in the template overrides previously installed SDK's.